Cargo.toml

```diff
@@ -103,19 +103,19 @@ members = [
 
 [workspace.dependencies]
 acpi_tables = { git = "https://github.com/rust-vmm/acpi_tables", branch = "main" }
-kvm-bindings = "0.9.1"
-kvm-ioctls = "0.18.0"
-linux-loader = "0.12.0"
+kvm-bindings = "0.10.0"
+kvm-ioctls = "0.19.0"
+linux-loader = "0.13.0"
 mshv-bindings = { git = "https://github.com/rust-vmm/mshv", tag = "v0.3.0" }
 mshv-ioctls = { git = "https://github.com/rust-vmm/mshv", tag = "v0.3.0" }
```

Comment:
While at it can you also update mshv-bindings and mshv-ioctls to `0.3.1`, since these are now published on crates.io and are also rust-vmm crates. https://crates.io/search?q=mshv

Reply:
Noted, I will take care of it 👌